content/en/users/cloud-compute/ec3/apps/_index.md

```diff
@@ -33,26 +33,44 @@ Check the
 To deploy a virtual cluster with
 [Marathon](https://mesosphere.github.io/marathon/),
 [Mesos](http://mesos.apache.org/), and
-[Chronos](https://mesos.github.io/chronos/) as an orchestration,
-please select Mesos + Marathon + Chronos from the list of available LRMS.
+[Chronos](https://mesos.github.io/chronos/) as an orchestration, please select
+Mesos + Marathon + Chronos from the list of available LRMS.
 
 ### OSCAR cluster
 
 To deploy
 [Serverless computing for data-processing applications](https://www.egi.eu/about/newsletters/serverless-computing-for-data-processing-applications-in-egi/)
 in EGI, please select OSCAR from the list of LRMS (Local Resource Management
-System). Use cases of applications that use the OSCAR framework for
-event-driven high-throughput processing of files:
-
-* [Plants Classification](https://github.com/indigo-dc/plant-classification-theano),
+System). OSCAR supports data-driven serverless computing for file-processing
+applications. A file upload, to an object storage backend, will trigger the
+execution of a chosen shell script running inside a user-defined container.
+These will be orchestrated as a Kubernetes batch jobs. The output data will be
+uploaded to any object storage backends support. Synchronous invocations
```

Comment:
```suggestion
uploaded to any object storage that the backends support. Synchronous invocations are
```

Still it's a bit fuzzy how the backend relates to the object storage, probably this is detailed in the OSCAR docs, but here it's confusing. Maybe this whole sentence should be simplified like (just a suggestion, please improve as needed) and the details can be then read at OSCAR's docs

```
File uploads to a supported object storage will trigger the execution of actions inside user-defined
containers that are orchestrated as Kubernetes batch jobs. Output of those actions can be uploaded 
to object storage as well.
```

Reply:
I have updated the description. Please check if it is clearer now.